src/main/java/com/minecolonies/colony/jobs/JobBuilder.java

```diff
@@ -159,20 +168,27 @@ public void setSchematic(SchematicWrapper schematic)
     }
 
     /**
-     * Returns the work interval of the worker //unfinished
+     * Returns the work interval of the worker.
      *
      * @return work interval
      */
     public int getWorkInterval()
     {
-        return 1;//Constants.BUILDERWORKINTERFALL - this.getLevel();//TODO
+        //TODO
+        //Constants.BUILDERWORKINTERFALL - this.getLevel();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=squid%3ACommentedOutCodeLine)